README.md

```diff
@@ -91,9 +87,25 @@ npm run test [component-name(s)]
 npm run pr
 ```
 
+### Documentation site
+Make sure you install the documentation site dependencies first.
```

Comment:
Let's add this as a postinstall task so users don't have to remember to do this.

Reply:
bump

Reply:
https://github.com/patternfly/patternfly-elements/pull/1508